web/models/stable_diffusion/stable_args.py

```diff
@@ -32,7 +32,7 @@
 p.add_argument(
     "--seed",
     type=int,
-    default=42,
+    default=-1,
```

Comment:
Why do we change the seed>

Reply:
actually seed does not matter for web as it is always supplied by UI, but -1 in general means random seed at each generation.